styx-scheduler-service/src/main/java/com/spotify/styx/docker/KubernetesPodEventTranslator.java

```diff
@@ -271,4 +271,14 @@ static boolean hasPullImageError(ContainerStatus cs) {
         || "ErrImagePull".equals(waiting.getReason())
         || "ImagePullBackOff".equals(waiting.getReason()));
   }
+
+  static boolean isTerminated(ContainerStatus cs) {
```

Comment:
Why are these two methods in this class?

Reply:
because `hasPullImageError` is here, seemed like a good fit